cloud-nio/cloud-nio-impl-drs/src/main/scala/cloud/nio/impl/drs/DrsPathResolver.scala

```diff
@@ -0,0 +1,85 @@
+package cloud.nio.impl.drs
+
+import com.typesafe.config.Config
+import org.apache.http.{HttpStatus, StatusLine}
+import org.apache.http.client.methods.{CloseableHttpResponse, HttpPost}
+import org.apache.http.entity.{ContentType, StringEntity}
+import org.apache.http.impl.client.{CloseableHttpClient, HttpClientBuilder}
+import org.apache.commons.lang3.exception.ExceptionUtils
+import org.apache.http.util.EntityUtils
+import spray.json._
```

Comment:
Ew. 😉 Consider circe in the future. It's the (latest) hotness.

Reply:
😄